src/main/java/greencity/service/impl/PlaceServiceImpl.java

```diff
@@ -0,0 +1,89 @@
+package greencity.service.impl;
+
+import greencity.entity.Place;
+import greencity.entity.enums.PlaceStatus;
+import greencity.exception.NotFoundException;
+import greencity.repository.PlaceRepo;
+import greencity.service.PlaceService;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+@Slf4j
+public class PlaceServiceImpl implements PlaceService {
+    private final PlaceRepo placeRepo;
+
+    /**
+     * Get current date and time in zone.
+     *
+     * @param zoneId - zone id.
+     * @return LocalDateTime object.
+     * @author Nazar Vladyka.
+     */
+    public static LocalDateTime getDateTime(String zoneId) {
```

Comment:
Maybe move this method to a DateTimeService because it does nothing related to Place

Reply:
done